feat: add libzenohcpp homebrew formula

diogomatsubara opened the pull request.

diogomatsubara committed:
feat: add libzenohcpp homebrew formula

DenisBiryukov91 commented on libzenohcpp.rb:
```diff
@@ -0,0 +1,79 @@
+    lib.install "lib/pkgconfig/zenohcxx.pc"
+    lib.install "lib/cmake/zenohcxx/zenohcxxConfig.cmake"
+    lib.install "lib/cmake/zenohcxx/zenohcxxConfigVersion.cmake"
+    include.install "include/zenoh.hxx"
```
is it possible not to list all the include files, since those are being added quite frequently, and instead just install the whole "include" folder ?

diogomatsubara replied:
I looked up some other examples and it seems to be possible but I haven't tested it yet. Pushed the change in 3c25eb7

diogomatsubara committed:
fix: install full include dir

This way we don't have to keep track of individual files that needs
installing.
